src/pages/gift/about/FAQ.js

```diff
@@ -10,11 +10,11 @@ export default function FAQ () {
   const q7 = 'I am having trouble claiming my gift.';
   const a1 = (
     <p className="pb-3">
-      Polkadot is a scalable, interoperable & secure network protocol for the
-      next web. DOT is the utility that serves different functions in the
-      network. You can find out more about Polkadot&nbsp;
+      {giftTheme.network} is a scalable, interoperable & secure network protocol for the
+      next web. {giftTheme.content} is the utility that serves different functions in the
```

Comment:
This template won't work for NFTs. It will replace the content as "NFT is the utility that serves different functions " We might want to have separate FAQ pages for each theme with static contents and then load each page based on the theme. like what we do for footers.

Reply:
For NFTs we're currently not having an About page, but linking to https://kusama.network/parachains/#claim-nft from the menu instead. Though if there'll be further different campaigns in the future, then having different FAQ sites makes sense.

Reply:
makes sense. LGTM.